<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5OULTSJ" failures="1" tests="3" name="Test Basics 2" time="0.049" errors="0" timestamp="13 Jun 2022 02:43:22 GMT">
  <testcase classname="Factory.Test2" name="test22mehtod" time="0.01"/>
  <testcase classname="Factory.Test2" name="test2method" time="0.001"/>
  <testcase classname="Factory.Test2" name="testFail" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Factory.Test2.testFail(Test2.java:24)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- testFail -->
</testsuite> <!-- Test Basics 2 -->
